packages/license-kit/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "license-kit",
+  "version": "0.1.2",
+  "description": "Aggregate license notes of OSS libraries used in your Node.js project",
+  "main": "build/index",
+  "module": "build/index",
+  "types": "build/index.d.ts",
+  "bin": "build/index.js",
+  "exports": {
+    ".": {
+      "source": "./src/index.ts",
+      "types": "./build/typescript/index.d.ts",
+      "default": "./build/module/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "src",
+    "build",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__",
+    "!**/.*"
+  ],
+  "scripts": {
+    "test": "jest",
+    "typecheck": "tsc",
+    "lint": "eslint \"**/*.{js,ts,tsx}\"",
+    "clean": "del-cli build",
+    "build-library": "tsc -p tsconfig.json"
+  },
+  "keywords": [
+    "nodejs",
+    "oss",
+    "license",
+    "acknowledgements",
+    "license-management"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/callstackincubator/react-native-legal.git",
+    "directory": "packages/license-kit"
+  },
+  "author": "Mateusz Mędrek <luigimario1913@gmail.com> (https://github.com/mateusz1913)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/callstackincubator/react-native-legal.git/issues"
+  },
+  "homepage": "https://callstackincubator.github.io/react-native-legal/",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.5",
+    "@types/minimist": "^1.2.5",
+    "del-cli": "^5.1.0",
```

Comment:
seems like it is only used in `clean` command - let's replace it with `rimraf` to decrease amount of dependecies

Reply:
Done in 9391aa3066c594d2aa6abd181b2533b3484ea973